dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use an official Python runtime as a parent image 
+FROM python:3.10-slim
+
+# Set environment variables 
+ENV PTYTHONDONTWRITEBYTECODE 1
+ENV PYTHONBUFFERED 1
+
+# Set the working directory in the container 
+WORKDIR /app
+
+# Copy the requirements file into the container 
+COPY requirements.txt /app/
+
+# Install any needed packages specified in the requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
@trojrobert thanks for this pr .. much needed! 
lida already specifies dependencies using pyproject.toml. and uses the build_backend is specified. 

as part of this requirements.txt references this same file by using . 
The following command in docker should install lida 

```bash
RUN pip install --no-cache-dir /app/lida 
``` 

I see that you have added a whole new requirements.txt and a using poetry build system. Any reasons why?

Reply:
@victordibia I had some problem with the dependencies because each dependency was trying to install the latest version. That was why I did that. I think the problem was the version of my poetry. I have corrected that